internal/services/network/application_gateway_resource.go

```diff
@@ -528,6 +599,79 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 				Set: applicationGatewayHttpListnerHash,
 			},
 
+			"listener": {
```

Comment:
Does this conflicts with `http_listener`?

Reply:
No, they can be configured together.